cloud-functions/utils/__init__.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import flask
+
+""" A Class to hold commonly used functions
+that can be reused by the different cloud functions
```

Comment:
nit: This is really just a collection of function in Python module, not a an actual class. I think this is totally fine, but we probably shouldn't call it a class.